packages/shopify-codemod/test/fixtures/convert-default-export-objects-to-named-exports/literals.input.js

```diff
@@ -0,0 +1,9 @@
+export default {
```

Comment:
Three sanity tests I'd like to see:

- default exports that do not export objects (as noted below)
- named exports that export an object (unchanged)
- objects with computed properties (will have to stay unchanged since we don't know the name of the export statically).